src/søknad/steg/7-oppsummering/OppsummeringBosituasjon.tsx

```diff
@@ -24,7 +25,7 @@ const OppsummeringBosituasionenDin: React.FC<Props> = ({ bosituasjon }) => {
 
   return (
     <Ekspanderbartpanel tittel={<Undertittel>Bosituasjonen din</Undertittel>}>
-      <div className="oppsummering-bosituasjon">
+      <EkspanderbarOppsummering>
```

Comment:
Skal denne ha ekspanderbarOppsummering?

Reply:
Mm, det var for å teste om vi ikke kunne erstatte less fila med en styled component wrapper, og tror vi kan det. Har glemt å erstatteden på de andre 🙈  Fikser

Reply:
aaah - lurt!